### ISSUE TITLE
Unable to boot Rei_Six from NAND without SD card

### ISSUE THREAD

**Issue by WafflesYem** [CrimsonMaple/Rei-Six#5]

**Comment by CrystaltheGlaceon** [comment]
Rei-Six requires the rei folder to boot, which can only be stored on the SD card to prevent accidentally locking the user from being able to launch payloads.

**Comment by CrimsonMaple** [comment]
Yeah there is an issue with Rei-Six not properly loading without loader on SD. I will get around to fixing it at some point.
